examples/demo-app/src/analytics.js

```diff
@@ -0,0 +1,53 @@
+import { ActionTypes } from "kepler.gl/actions";
+
+import { LOCATION_CHANGE } from "react-router-redux";
+
+const {
+  ADD_DATA,
+  LAYER_CONFIG_CHANGE,
+  LAYER_HOVER,
+  MAP_STYLE_CHANGE,
+  UPDATE_MAP,
+  TOGGLE_MODAL,
+  TOGGLE_SIDE_PANEL
+} = ActionTypes;
+
+const getPayload = ({ payload }) => payload;
+
+const trackingInformation = {
+  [ADD_DATA]: ({ payload: { files } }) =>
```

Comment:
`ADD_DATA` is not used, `LOAD_FILES` is the one you want to track

Reply:
Yep!